README.md

```diff
@@ -21,6 +21,7 @@ Optuna-Integration API reference is [here](https://optuna-integration.readthedoc
 * [Chainer](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#chainer)  ([example](https://github.com/optuna/optuna-examples/tree/main/chainer/chainer_integration.py))
 * [ChainerMN](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#chainermn)  ([example](https://github.com/optuna/optuna-examples/tree/main/chainer/chainermn_simple.py))
 * [Keras](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#keras)  ([example](https://github.com/optuna/optuna-examples/tree/main/keras))
+* [skorch](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#skorch)
```

Comment:
An example of `skorch` is [here](https://github.com/optuna/optuna-examples/blob/6d3fedcadc67c3f81b882fad9dffd28612967a0c/pytorch/skorch_simple.py).

Reply:
I have added the link.